### ISSUE TITLE
Add UXG Max gateway to supported gateway list

### ISSUE THREAD

**Issue by exzz** [unpoller/dashboards#36]
Hello,

Many thanks for all this work.

New released "Gateway Max" identified as type 'uxg', compared to my previous model, 'usg'. 
Grafana variables need to be updated to show related metrics in USG dashboard:
```
Controller --> label_values(unpoller_device_info{type=~"udm|usg|ugw|uxg"},source)	
Site --> label_values(unpoller_device_info{source=~"$Controller", type=~"udm|usg|ugw|uxg"},site_name)	
Gateway --> label_values(unpoller_device_info{site_name=~"$Site", type=~"udm|usg|ugw|uxg"},name)	
```

I guess it is the same for new "UniFi Cloud Gateway Ultra" that may match type = 'ucg' (I do not own such device to validated the type value)

**Comment by CollinLeishman** [comment]
I came here to say the same thing. All that needs to be done is add uxg to all instances of `type=~\"udm|usg|ugw\" `

**Comment by platinummonkey** [comment]
Fixed in #41
